### PR TITLE
feat: add badge atomic component

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-507

I changed ``badgeContent`` prop name to ``content``
Added ``dataActionId`` prop
Instead of ``backgroundColor`` and ``color`` used a badgeStyle type

let me know if any of these doesn't make sense

### CODE REVIEW

File: packages/ui/src/1_atoms/Badge/Badge.module.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.badge {
+  @apply rounded-full inline-flex items-center justify-center overflow-hidden;
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Badge/Badge.module.css]:
```suggestion
  @apply rounded-full inline-flex items-center justify-center overflow-hidden font-roboto not-italic;
```

File: packages/ui/src/1_atoms/Badge/Badge.module.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  font-family: 'Roboto';
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Badge/Badge.module.css]:
```suggestion

```

File: packages/ui/src/1_atoms/Badge/Badge.module.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  font-style: normal;
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Badge/Badge.module.css]:
```suggestion

```

File: packages/ui/src/1_atoms/Badge/Badge.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+export const Badge: FC<PropsWithChildren<BadgeProps>> = ({
+  content,
+  className,
+  style = BadgeStyle.gray,
+  size = BadgeSize.sm,
+  dataActionId,
+}) => {
+  return (
+    <span
+      className={classNames(
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Badge/Badge.tsx:25]: i would suggest using useMemo here for classes

File: packages/ui/src/1_atoms/Badge/Badge.module.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+  &.sm {
+    @apply h-3.5 w-3.5 text-tiny font-medium;
+  }
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Badge/Badge.module.css:6]:
```suggestion
  }
  
```

**Review by creed-victor — APPROVED** [review]

**Review by soulBit — APPROVED** [review]

**Review by LunaSov — APPROVED** [review]

**Pull request merged**